src/app/components/challenge/challengesubmit/challengesubmit.component.ts

```diff
@@ -394,12 +394,20 @@ export class ChallengesubmitComponent implements OnInit {
   formSubmit(self) {
     self.submissionError = '';
     const submissionFile = self.globalService.formItemForLabel(self.components, 'input_file').fileValue;
+    const submissionProjectUrl = self.globalService.formValueForLabel(self.components, 'project_url');
+    const submissionPublicationUrl = self.globalService.formValueForLabel(self.components, 'publication_url');
     if (submissionFile === null || submissionFile === '') {
       self.submissionError = 'Please upload file!';
       return;
     } else if (self.selectedPhase['id'] === undefined) {
       self.submissionError = 'Please select phase!';
       return;
+    } else if (submissionProjectUrl == null || submissionProjectUrl === '') {
+        self.submissionError = 'Please provide a valid project url!';
```

Comment:
@robingan7 can you fix the indentation. Here you are using 4 spaces but code above uses 2 spaces to indent. Please make this consisent